src/sidebar/widget-controller.js

```diff
@@ -135,6 +135,12 @@ module.exports = function WidgetController(
       $scope.$evalAsync(function () {
         searchClients.splice(searchClients.indexOf(searchClient), 1);
       });
+
+      annotationUI.frames().forEach(function (frame) {
+        if (0 <= uris.indexOf(frame.uri)) {
```

Comment:
By convention, we tend to write these checks in the form `expression <relation> <constant value>`, eg. `uris.indexOf(frame.uri) >= 0`. I think that tends to be a little quicker to read.